src/vivarium_census_prl_synth_pop/constants/data_values.py

```diff
@@ -92,6 +89,12 @@ class MilitaryEmployer(NamedTuple):
     PROPORTION_WORKFORCE_EMPLOYED = 0.0032
 
 
+class Unemployed(NamedTuple):
+    EMPLOYER_ID = 0
+    EMPLOYER_ADDRESS_ID = 0
+    EMPLOYER_NAME = "unemployed"
```

Comment:
Not for this PR, but this column should actually be a name id, not a name

Reply:
Yeah i figured. For now I just kinda stripped out the name from the address tests; name tests will need to be written when employer_name_id gets implemented